dll/kernel32.cpp

```diff
@@ -1823,6 +1823,10 @@ namespace kernel32 {
 		}
 	}
 
+	int WIN_FUNC GetLocaleInfoW(unsigned int Locale, int LCType, char *lpLCData, int cchData) {
+		return GetLocaleInfoA(Locale, LCType, lpLCData, cchData); // lol
```

Comment:
You're going to need to do a wide string conversion of lpLCData for this function to make any sense. Otherwise you might as well stub it

Reply:
how about now?